common/source_model/src/main/scala/weco/catalogue/source_model/sierra/rules/SierraItemAccess.scala

```diff
@@ -276,6 +276,20 @@ object SierraItemAccess extends SierraQueryOps with Logging {
           note = Some(message)
         )
 
+      //
+      case (
+            _,
+            Some(Status.Safeguarded),
+            Some(OpacMsg.ByApproval),
```

Comment:
Is the correct opac message a requirement to result in this access condition? Can we safely omit it and rely solely on the status? I'm guessing the answer is _yes_ and i've missed some context, so not blocking.

Reply:
No this is a good question - I put the opac message in initially because we match on both for all the other access statuses. I thought about it a bit more and I reckon the reason we do that is so that we control very tightly on how we map access statuses; the opac message and the status should always come together and so if they don't I think we'd regard it as a data issue.